doc/development/tutorials/examples/todo.py

```diff
@@ -90,27 +89,28 @@ def process_todo_nodes(app, doctree, fromdocname):
         for todo_info in env.todo_all_todos:
             para = nodes.paragraph()
             filename = env.doc2path(todo_info['docname'], base=None)
-            description = (
-                _('(The original entry is located in %s, line %d and can be found ') %
-                (filename, todo_info['lineno']))
+            description = _(
+                '(The original entry is located in %s, line %d and can be found '
+            ) % (filename, todo_info['lineno'])
             para += nodes.Text(description)
 
             # Create a reference
             newnode = nodes.reference('', '')
             innernode = nodes.emphasis(_('here'), _('here'))
             newnode['refdocname'] = todo_info['docname']
-            newnode['refuri'] = app.builder.get_relative_uri(
-                fromdocname, todo_info['docname'])
+            newnode['refuri'] = app.builder.get_relative_uri(fromdocname, todo_info['docname'])
             newnode['refuri'] += '#' + todo_info['target']['refid']
             newnode.append(innernode)
             para += newnode
             para += nodes.Text('.)')
 
             # Insert into the todolist
-            content.extend((
-                todo_info['todo'],
-                para,
-            ))
+            content.extend(
```

Comment:
For instance, here, I would have expected ruff to use a single line when possible (I feel we lose more with this format...)

Reply:
Nope, if it does not fit on a line of length <= 95 (what is set in `.ruff.toml`) then it will do this

> I feel we lose more with this format.

personally I disagree, its just formatting that no one should have to care about.
Or, if you do, open an issue on ruff 😬

Reply:
i agree with @chrisjsewell that the point of autoformatting is not to get exactly the formatting you want, it's to never think about formatting again.

That said, the `--preview` flag would reduce the indentation for this specific case

Reply:
As above, added `preview = true` and the change is now gone